examples/va/nfv/ovs-dpdk/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenStack OvS DPDK
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/12c57baeca4ae33dd30a7707d330eb094309b4cd) on Jan 31st, 2024**
+
+This is a collection of CR templates that represent a validated Red Hat OpenStack Services on OpenShift deployment that has the following characteristics:
+
+- 3 master/worker combo-node OpenShift cluster
```

Comment:
We could put OvS DPDK on this list but I guess we're assuming that's obvious given the title of the document.